src/app/report/report.component.ts

```diff
@@ -77,7 +77,7 @@ export class ReportComponent implements AfterViewInit, OnInit {
     if (this.newTab && this.reportData) {
       this.tabService.closeTab(this.reportData);
     }
-    this.displayComponent.closeReport(false);
+    this.displayComponent.closeReport();
   }
 
   closeReport(): void {
```

Comment:
closeReport is not used anymore. Is there still a reason to keep this?

Reply:
We don't remove one single report from the debug tree, right? If it's no, then there is no reason I think. What do you think @MatthijsSmets?

Reply:
You can remove this method